ext/soap/soap.stub.php

```diff
@@ -13,19 +13,19 @@ public function __construct(mixed $data, string $name) {}
 
 class SoapHeader
 {
-    public function __construct(string $namespace, string $name, mixed $data = UNKNOWN, bool $mustunderstand = false, string|int|null $actor = null) {}
+    public function __construct(string $namespace, string $name, mixed $data = UNKNOWN, bool $mustUnderstand = false, string|int|null $actor = null) {}
 }
 
 class SoapFault extends Exception
 {
-    public function __construct(array|string|null $faultcode, string $faultstring, ?string $faultactor = null, mixed $detail = null, ?string $faultname = null, mixed $headerfault = null) {}
+    public function __construct(array|string|null $code, string $string, ?string $actor = null, mixed $details = null, ?string $name = null, mixed $headerFault = null) {}
```

Comment:
Struggled with this one ... the problem is that the previous names match the property names on SoapFault. However, the old param names on the SoapVar ctor don't match the property names, so in the end I went with renaming them.